packages/suite/src/components/suite/ChangeDeviceLabel.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from 'react';
+import { analytics, EventType } from '@trezor/suite-analytics';
+import styled, { css } from 'styled-components';
+
+import { Button, INPUT_HEIGHTS, Input } from '@trezor/components';
+import { Translation } from 'src/components/suite';
+import { useDevice, useDispatch, useTranslation } from 'src/hooks/suite';
+import * as deviceSettingsActions from 'src/actions/settings/deviceSettingsActions';
```

Comment:
This import pattern was used because of `useActions`, we can now simply do:

```suggestion
import { applySeetings } from 'src/actions/settings/deviceSettingsActions';
```